src/source/Crypto/Tls_mbedtls.c

```diff
@@ -4,6 +4,35 @@
 #define LOG_CLASS "TLS_mbedtls"
 #include "../Include_i.h"
 
+// Read and parse CA certificate
+PRIVATE_API STATUS readAndParseCACertificate(PTlsSession pTlsSession)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    UINT64 cert_len = 0;
+    PBYTE cert_buf = NULL;
+    CHAR errBuf[128];
+
+    CHK_STATUS(readFile(KVS_CA_CERT_PATH, FALSE, NULL, &cert_len));
+    CHK(cert_len > 0, STATUS_INVALID_CERT_PATH_LENGTH);
+    cert_buf = (PBYTE) MEMCALLOC(1, cert_len + 1);
+    CHK(cert_buf != NULL, STATUS_NOT_ENOUGH_MEMORY);
+    CHK_STATUS(readFile(KVS_CA_CERT_PATH, FALSE, cert_buf, &cert_len));
+    int ret = mbedtls_x509_crt_parse(&pTlsSession->cacert, cert_buf, (size_t)(cert_len + 1));
```

Comment:
Suggest to use the PIC definitions for the types (INT32, SIZE_T)

Reply:
Hi @sirknightj this is intentional to match the mbedtls API return type and the `size_t` type for length.

Reply:
@sirknightj @unicornss  changed size_t to AWS define SIZE_T. `int`, however, should be kept as is, as there is no define for the same.